static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-banner-editorial-independence.js

```diff
@@ -0,0 +1,41 @@
+// @flow strict
+import { makeBannerABTestVariants } from 'common/modules/commercial/contributions-utilities';
+
+const componentType: OphanComponentType = 'ACQUISITIONS_ENGAGEMENT_BANNER';
+const abTestName: string = 'AcquisitionsBannerEditorialIndependence';
+
+const engagementBannerCopyTest = (): string =>
```

Comment:
is there any need for this to be a function?

Reply:
No, changed to string.